### PR TITLE
fix log duplication

### PR DESCRIPTION
**Pull request by bendaamerahm**

### CODE REVIEW

File: src/test/java/org/gridsuite/modification/server/ModificationControllerTest.java [hunk 1]
```diff
@@ -2646,10 +2646,8 @@ public void testLineAttachToSplitLine() throws Exception {
         LinesAttachToSplitLinesInfos linesAttachToAbsentLine1 = new LinesAttachToSplitLinesInfos("absent_line_id", "line2", "line3", "v4", "1.A", "nl4", "NewLine4", "nl5", "NewLine4");
 
         String linesAttachToAbsentLine1Json = objectWriter.writeValueAsString(linesAttachToAbsentLine1);
-        mvcResult = mockMvc.perform(post(linesAttachToSplitLinesUriString, TEST_NETWORK_ID).content(linesAttachToAbsentLine1Json).contentType(MediaType.APPLICATION_JSON))
-                .andExpect(status().is4xxClientError()).andReturn();
-        resultAsString = mvcResult.getResponse().getContentAsString();
-        assertEquals(resultAsString, String.format("LINE_NOT_FOUND : Line %s is not found", "absent_line_id"));
+        mockMvc.perform(post(linesAttachToSplitLinesUriString, TEST_NETWORK_ID).content(linesAttachToAbsentLine1Json).contentType(MediaType.APPLICATION_JSON))
+                .andExpect(status().isOk()).andReturn();
```
Comment (FranckLecuyer) [inline src/test/java/org/gridsuite/modification/server/ModificationControllerTest.java:2651]: Maybe you can test that nothing has changed on the network : the three existing lines are already there, the new line has not been created, the voltage levels have not been deleted ....
In order to replace the assertion that was previously done on the exception error message ...

**Review by FranckLecuyer — CHANGES_REQUESTED** [review]

File: src/test/java/org/gridsuite/modification/server/ModificationControllerTest.java (hunk 1 above)
Comment (bendaamerahm) [reply src/test/java/org/gridsuite/modification/server/ModificationControllerTest.java:2664]: nice catch

File: src/test/java/org/gridsuite/modification/server/ModificationControllerTest.java [hunk 2]
```diff
@@ -2638,16 +2640,27 @@ public void testLineAttachToVoltageLevel() throws Exception {
     }
 
     @Test
-    public void testLineAttachToSplitLine() throws Exception {
+    public void testAttachLinesToSplitLines() throws Exception {
         MvcResult mvcResult;
         String resultAsString;
         String linesAttachToSplitLinesUriString = "/v1/networks/{networkUuid}/lines-attach-to-split-lines?group=" + TEST_GROUP_ID + "&reportUuid=" + TEST_REPORT_ID;
-
+        networkStoreService.getNetwork(TEST_NETWORK_ID);
```
Comment (FranckLecuyer) [inline src/test/java/org/gridsuite/modification/server/ModificationControllerTest.java:2647]: useless line

File: src/test/java/org/gridsuite/modification/server/ModificationControllerTest.java [hunk 3]
```diff
@@ -2638,16 +2640,27 @@ public void testLineAttachToVoltageLevel() throws Exception {
...
 
         String linesAttachToAbsentLine1Json = objectWriter.writeValueAsString(linesAttachToAbsentLine1);
+
+        Line prevL2 = network.getLine("line2");
+        Line prevL3 = network.getLine("line3");
+        VoltageLevel prevVl = network.getVoltageLevel("v4");
+
         mockMvc.perform(post(linesAttachToSplitLinesUriString, TEST_NETWORK_ID).content(linesAttachToAbsentLine1Json).contentType(MediaType.APPLICATION_JSON))
                 .andExpect(status().isOk()).andReturn();
+        assertEquals(prevL2.getId(), network.getLine("line2").getId());
```
Comment (FranckLecuyer) [inline src/test/java/org/gridsuite/modification/server/ModificationControllerTest.java:2658]: assertNotNull(network.getLine("line2"));

File: src/test/java/org/gridsuite/modification/server/ModificationControllerTest.java [hunk 4]
```diff
@@ -2638,16 +2640,27 @@ public void testLineAttachToVoltageLevel() throws Exception {
...
+        assertEquals(prevL3.getId(), network.getLine("line3").getId());
```
Comment (FranckLecuyer) [inline src/test/java/org/gridsuite/modification/server/ModificationControllerTest.java:2659]: assertNotNull(network.getLine("line3"));

File: src/test/java/org/gridsuite/modification/server/ModificationControllerTest.java [hunk 5]
```diff
@@ -2638,16 +2640,27 @@ public void testLineAttachToVoltageLevel() throws Exception {
...
+        assertNull(network.getLine("absent_line_id"));
+        assertEquals(prevVl.getId(), network.getVoltageLevel("v4").getId());
```
Comment (FranckLecuyer) [inline src/test/java/org/gridsuite/modification/server/ModificationControllerTest.java:2661]: assertNotNull(network.getVoltageLevel("v4"));

File: src/test/java/org/gridsuite/modification/server/ModificationControllerTest.java [hunk 6]
```diff
@@ -2638,16 +2640,27 @@ public void testLineAttachToVoltageLevel() throws Exception {
...
         String linesAttachToSplitLinesUriString = "/v1/networks/{networkUuid}/lines-attach-to-split-lines?group=" + TEST_GROUP_ID + "&reportUuid=" + TEST_REPORT_ID;
-
+        networkStoreService.getNetwork(TEST_NETWORK_ID);
         LinesAttachToSplitLinesInfos linesAttachToAbsentLine1 = new LinesAttachToSplitLinesInfos("absent_line_id", "line2", "line3", "v4", "1.A", "nl4", "NewLine4", "nl5", "NewLine4");
 
         String linesAttachToAbsentLine1Json = objectWriter.writeValueAsString(linesAttachToAbsentLine1);
+
+        Line prevL2 = network.getLine("line2");
+        Line prevL3 = network.getLine("line3");
+        VoltageLevel prevVl = network.getVoltageLevel("v4");
```
Comment (FranckLecuyer) [inline src/test/java/org/gridsuite/modification/server/ModificationControllerTest.java:2654]: Remove these 3 lines

**Review by FranckLecuyer — CHANGES_REQUESTED** [review]

**Review by FranckLecuyer — APPROVED** [review]

**Pull request merged**